### PR TITLE
Display Timezones in Admin 

### PR DESCRIPTION
**Pull request by lbwexler**
NOTE: This change needs the related Hoist-Core update to see the timezones, but should be backward compatible.

Hoist P/R Checklist
-------------------

**Pull request authors:** Review and check off the below. Items that do not apply can also be
checked off to indicate they have been considered. If unclear if a step is relevant, please leave
unchecked and note in comments.

- [x] Caught up with `develop` branch as of last change.
- [x] Added CHANGELOG entry, or determined not required.
- [x] Reviewed for breaking changes, added `breaking-change` label + CHANGELOG if so.
- [x] Updated doc comments / prop-types, or determined not required.
- [x] Reviewed and tested on Mobile, or determined not required.
- [x] Created Toolbox branch / PR, or determined not required.

**If your change is still a WIP**, please use the "Create draft pull request" option in the split
button below to indicate it is not ready yet for a final review.

> **Pull request reviewers:** when merging this P/R, please consider using a *squash commit* to
> collapse multiple intermediate commits into a single commit representing the overall feature
> change. This helps keep the commit log clean and easy to scan across releases. PRs containing a
> single commit should be *rebased* when possible.

### CODE REVIEW

File: admin/tabs/general/about/AboutPanel.js [hunk 1]
```diff
@@ -41,30 +43,33 @@ function renderTables() {
...
-                row('Environment', svc.get('appEnvironment')),
-                row('Database', svc.get('databaseConnectionString')),
-                row('DB User / Create Mode', `${svc.get('databaseUser')} / ${svc.get('databaseCreateMode')}`),
+                row('App Name / Code', `${get('appName')} / ${get('appCode')}`),
+                row('Environment', get('appEnvironment')),
+                row('Database', get('databaseConnectionString')),
+                row('DB User / Create Mode', `${get('databaseUser')} / ${get('databaseCreateMode')}`),
+                row('App Timezone', fmtTimezone(get('appTimezone'), get('appTimezoneOffset'))),
+                row('Server Timezone', fmtTimezone(get('serverTimezone'), get('serverTimezoneOffset'))),
+                row('Client Timezone',  fmtTimezone('Unknown', (new Date()).getTimezoneOffset()*MINUTES*-1)),
```
Comment (amcclain) [inline admin/tabs/general/about/AboutPanel.js:52]: Note we could use `Intl.DateTimeFormat().resolvedOptions().timeZone` if we wanted the display the "Americas/Los Angeles" form of the client TZ alongside the more descriptive zone names for server/app.

(Also note the capital Z in that API.)

File: admin/tabs/general/about/AboutPanel.js [hunk 2]
```diff
@@ -24,15 +26,15 @@ export const aboutPanel = hoistCmp.factory(
 );
 
 function renderTables() {
-    const svc = XH.environmentService,
-        startupTime = svc.get('startupTime'),
+    const get = (str) => XH.environmentService.get(str),
```
Comment (amcclain) [inline admin/tabs/general/about/AboutPanel.js:29]: 👍  😃

File: cmp/layout/Tags.js [hunk 3]
```diff
@@ -42,4 +42,5 @@ export const textarea = elemFactory('textarea');
 export const th = elemFactory('th');
 export const thead = elemFactory('thead');
 export const tr = elemFactory('tr');
-export const ul = elemFactory('ul');
\ No newline at end of file
+export const ul = elemFactory('ul');
+export const b = elemFactory('b');
```
Comment (amcclain) [inline cmp/layout/Tags.js:46]: Nit - could we use `strong` instead, or `span({className: 'xh-bold'})`?  

(See https://developer.mozilla.org/en-US/docs/Web/HTML/Element/b re `b` as tag of last resort.)

File: admin/tabs/general/about/AboutPanel.js (hunk 1 above)
Comment (lbwexler) [reply admin/tabs/general/about/AboutPanel.js:52]: went with the capital Z on server and client -- think that makes sense(although our current Clock component is using the small Z as the JavaScript Date API....)

Fortunately, moment uses `tz` escaping the issue largely!

File: cmp/layout/Tags.js (hunk 3 above)
Comment (lbwexler) [reply cmp/layout/Tags.js:46]: 10-4

**Pull request merged**